Sources/Soto/Extensions/DynamoDB/DynamoDB+Codable.swift

```diff
@@ -221,11 +228,9 @@ extension DynamoDB {
         }
     }
 
-    public struct UpdateItemCodableInput<T: Encodable & Sendable>: AWSEncodableShape {
-        /// In case expressionAttributeNames is nil, the content of additionalAttributeNames is merged with the generated expressionAttributeNames dictionary and will override existing values. This can be used to specify complex conditionExpression with not auto-generated attribute names.
-        public let additionalAttributeNames: [String: String]?
+    public struct UpdateItemCodableInput<T: Encodable & Sendable, Additional: Encodable & Sendable>: AWSEncodableShape {
         /// The content of additionalAttributeValues is merged with expressionAttributeValues dictionary and will override existing values. This can be used to specify complex conditionExpression with not auto-generated attribute values.
-        public let additionalAttributeValues: [String: AttributeValue]?
```

Comment:
This is an API break isn't it?

Reply:
Yes it is. I can't see a better way to do this without crippling the API. Given the way people use the Request structures I thought I could get away. I will obviously report the break in the release notes